docs/API.md

```diff
@@ -177,31 +182,66 @@ const app = methods({
   POST: createUser
 })
 
-http.createServer(mount(app)).listen(3000)
+http.createServer(mount({ app })).listen(3000)
 ```
 
 ### mount
 
 ```haskell
-mount :: ((Request -> (Response | Promise Response)), Object) -> (IncomingMessage, ServerResponse) -> ()
+mount :: { k: v } -> (IncomingMessage, ServerResponse) -> ()
 ```
 
-Wraps a top-level handler function to prepare for mounting as a new `http` server.  Lifts the handler into a `Promise` chain, so the handler can respond with either a [`Response`](https://github.com/articulate/paperplane/blob/master/docs/getting-started.md#response-object), or a `Promise` that resolves with one.  Also accepts an options object with `errLogger` and `logger` properties, both of which can be set to [`logger`](#logger).
+Wraps a top-level handler function to prepare for mounting as a new `http` server.  Lifts the handler into a `Promise` chain, so the handler can respond with either a synchronous [`Response`](https://github.com/articulate/paperplane/blob/master/docs/getting-started.md#response-object), or a `Promise` or other ADT that resolves with one.  Accepts the following options:
+
+
+| Option | Type | Default | Description |
+| ------ | ---- | ------- | ----------- |
+| `app` | `Request -> m Response` | [`R.identity`](http://devdocs.io/ramda/index#identity) | top-level request handler function |
+| `cry` | `Error -> a` | [`paperplane.logger`](#logger) | error logger |
```

Comment:
Not to be a spoil-sport, but why the change from `errLogger` to `cry`?

`errLogger` is a lot clearer to me in terms of readability/understanding the intended usage without reading the docs.

Reply:
`cry` is a long-standing idiom on the Rise team for "error reporting", and is already a [function name exported by `glue`](https://github.com/articulate/glue/blob/master/API.md#cry).  I've opted to piggy-back on that custom to borrow the existing connotation.  (And it's likely that `glue#cry` will be dropped in favor of `paperplane-airbrake`, although `glue` will still setup our common Airbrake filters.)

Furthermore, the functionality is changing, since `cry` is intended to be the correct way to report errors to an aggregation service, not just log errors like `errLogger` did.  I thought the functionality change would be less confusing if the name also changed, to prevent a misunderstanding in the purpose.

It's also shorter, saving six whole keystrokes, not to mention that long pinky-stretch to the shift-key.  Your fingers will thank me later.  :trollface: